rescript/get_data.py

```diff
@@ -12,13 +12,132 @@
 import shutil
 import gzip
 import warnings
+import requests
+import tarfile
 
 import qiime2
 from urllib.request import urlretrieve
 from urllib.error import HTTPError
 from q2_types.feature_data import RNAFASTAFormat
 
 
+def _unite_dois_to_urls(DOIs):
+    '''Generate UNITE urls, given their DOIs.'''
+    # Make DOIs iterable
+    DOIs = [DOIs] if isinstance(DOIs, str) else DOIs
+    print('Get URLs for these DOIs:', DOIs)
+    base_url = 'https://api.plutof.ut.ee/v1/public/dois/'\
+               '?format=vnd.api%2Bjson&identifier='
+    # Eventual output
+    URLs = set()
+    # For each DOI, get download URL of file
+    for DOI in DOIs:
+        query_data = requests.get(base_url + DOI).json()
```

Comment:
why not use `urllib`, which is already imported above?

Reply:
ncbi.py also imports `requests`, and I found examples of using it to parse JSON.

What do you recommend?

Reply:
okay just ignore me! If RESCRIPt already has requests as a dependency then it's fine to use here of course.